File: js/bridge/SimpleDisplay.java
package bridge;

public class SimpleDisplay implements Display{
    @Override
    public void title(Draft draft) {
        System.out.println(draft.getTitle());
    }

    @Override
    public void author(Draft draft) {
        System.out.println(draft.getAuthor());
    }

    @Override
    public void content(Draft draft) {
        String[] contents = draft.getContents();
        for(String content: contents) {
            System.out.println(content);
        }
    }
}

File: ts/src/bridge/SimpleDisplay.ts
import Display from "./Display";
import Draft from "./Draft";

export default class SimpleDisplay implements Display {
  title(draft: Draft): void {
    console.log(draft.getTitle());
  }
  author(draft: Draft): void {
    console.log(draft.getTitle());
  }
  content(draft: Draft): void {
    const content = draft.getContent();
    content.forEach((item) => {
      console.log(item);
    });
  }
}
